src/sumo_store_riak.erl

```diff
@@ -38,6 +38,10 @@
 
 -include_lib("riakc/include/riakc.hrl").
 
+%% @todo it is needed with basho/riak-erlang-client 2.5.3. They already fixed on master so we should
```

Comment:
TODOs should start with a verb, like:
```erlang
%% @todo remove this when riakc … etc. etc.
```

The explanation can go below that line.